content/id/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -0,0 +1,94 @@
+weight: 30
+---
+
+{{% capture overview %}}
+
+Kubernetes mendukung banyak kluster virtual di dalam satu kluster fisik. kluster virtual tersebut disebut dengan *namespace*.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Kapan menggunakan banyak Namespace
+
+*Namespace* dibuat untuk digunakan di *environment* dengan banyak pengguna yang berada di dalam banyak tim ataupun proyek. Untuk sebuah kluster dengan beberapa pengguna saja, kamu tidak harus membuat ataupun memikirkan tentang *namespace*. Mulai gunakan *namespace* saat kamu membutuhkan fitur dari *namespace* itu sendiri.
+
+*Namespace* menyediakan ruang untuk nama objek. Nama dari *resource* atau objek harus berbeda di dalam sebuah *namespace*, tetapi boleh sama jika berbeda *namespace*. *Namespace* tidak bisa dibuat di dalam *namespace* lain dan setiap *resource* atau objek Kubernetes hanya dapat berada di dalam satu *namespace*.
+
+*Namespace* merupakan cara yang digunakan untuk memisahkan *resource* kluster untuk beberapa pengguna (dengan [*resource quota*](/docs/concepts/policy/resource-quotas/)).
+
+Dalam versi Kubernetes yang akan datang, objek di dalam satu *namespace* akan mempunyai *access control policies* yang sama secara *default*.
+
+Tidak perlu menggunakan banyak *namespace* hanya untuk memisahkan sedikit perbedaan pada *resource*, seperti perbedaan versi dari perangkat lunak yang sama: gunakan [labels](/docs/user-guide/labels) untuk membedakan *resource* di dalam *namespace* yang sama.
+
+## Bekerja dengan Namespace
+
+Pembuatan dan penghapusan *namespace* dijelaskan di [dokumentasi panduan admin untuk *namespace*](/docs/admin/namespaces).
+
+
+### Melihat namespace
+
+Kamu dapat melihat daftar *namespace* di dalam kluster menggunakan:
+
+```shell
+kubectl get namespace
+```
+```
+NAME          STATUS    AGE
+default       Active    1d
+kube-system   Active    1d
+kube-public   Active    1d
+```
+
+Kubernetes berjalan dengan tiga *namespace* awal:
+
+* `default`, *namespace default* untuk objek yang dibuat tanpa mencantumkan *namespace* pada spesifikasinya.
+* `kube-system`, *namespace* yang digunakan untuk objek yang dibuat oleh sistem Kubernetes.
+* `kube-public`, *namespace* ini dibuat secara otomatis dan dapat diakses oleh semua pengguna (termasuk yang tidak diutentikasi). *Namespace* ini disediakan untuk penggunaan kluster, jika beberapa *resouce* harus terlihat dan dapat dibaca secara publik di seluruh kluster. Aspek publik dari *namespace* ini hanya sebuah konvensi, bukan persyaratan.
+
+
+### Mengkonfigurasi namespace untuk request
+
+Untuk mengkonfigurasi sementara *request* untuk menggunakan *namespace* tertentu, gunakan `--namespace` *flag*.
+
+Sebagai contoh:
+
+```shell
+kubectl --namespace=<insert-namespace-name-here> run nginx --image=nginx
+kubectl --namespace=<insert-namespace-name-here> get pods
+```
+
+### Mengkonfigurasi preferensi namespace 
+
+Kamu dapat menyimpan konfigurasi *namespace* untuk semua perintah *kubectl* dengan perintah:
```

Comment:
should use backtick for `kubectl` (use backtick for command, fields, yaml, etc.)